script/Ciel/Day 5/05_01_CIEL05_4D.txt

```diff
@@ -217,63 +263,73 @@ Her surprise is completely understandable.
 [sha:31f091e0264cd81b49c0468dee0b5ac17f103f55]{
 -- Page 804, Offset 13028.
 -- 少年の名は教会の人間にとって無視できない事柄だ。
-His name is not something that can be ignored by the people of the Church.
+The boy's name is not something that can be taken lightly by members of the Church.
 }
 [sha:65d444335f65ff1baaac58b8abb00a2e47e112cb]{
 -- Page 804, Offset 13029.
 -- 聖堂教会の頂点である法王と、
-The Pope, the pinnacle of the Holy Church.
+The Pope is the pinnacle of the Holy Church.
 }
 [sha:cf570b2b5da72eed3a990720850659578e49f0d1]{
 -- Page 804, Offset 13030.
 -- その法王を支える128人の枢機卿。
-As well as the 128 Cardinals assisting him.
+Assisting him are 128 cardinals.
```

Comment:
as much as I love the power of 2 cardinal count, i think there's way more, mister

Reply:
Just checking this is a meta comment not a TL comment? You got me worried

Reply:
meta-commentary